be/src/vec/exec/vaggregation_node.h

```diff
@@ -157,7 +157,7 @@ struct AggregationMethodStringNoCache {
 
 /// For the case where there is one numeric key.
 /// FieldType is UInt8/16/32/64 for any type with corresponding bit width.
-template <typename FieldType, typename TData, bool consecutive_keys_optimization = true>
+template <typename FieldType, typename TData, bool consecutive_keys_optimization = false>
```

Comment:
Why set consecutive_keys_optimization = false here?

Reply:
Restricted to phmap's value_type contains const field(key), the cached value cannot be updated.